cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,117 @@
 /// <reference types='cypress' />
+const user = {
+  firstName: 'inna',
+  lastName: 'levytska',
+  email: 'inna123@gmail.com',
+  age: '40',
+  salary: '100',
+  department: 'QA'
+}
+beforeEach(() => {
+  cy.visit('https://demoqa.com/webtables')
+  });
 
 describe('Web Tables page', () => {
   it('', () => {
 
+
+  });
+  it('should have pagination', () => {
+    cy.get('.-pagination')
+      .should('contain', 'Previous')
+      .and('contain', 'Next');
+    cy.contains('.-pageInfo', 'Page')
+      .should('exist');
+  });
+  it('should have row count selection', () => {
+    cy.get('select')
+      .select('50 rows');
+  });
+  it('should provide abbility to add a new worker', () => {
+    cy.createWorker(user);
+    cy.get('.rt-tbody')
+      .should('contain', user.firstName)
+      .and('contain', user.lastName)
+      .and('contain', user.email);
+    cy.get('.rt-tbody')
+      .should('contain', user.age);
+    cy.get('.rt-tbody')
+      .should('contain', user.salary);
+    cy.get('.rt-tbody')
+      .should('contain', user.department);
+    cy.get('#searchBox')
+      .clear()
+      .type(user.firstName);
+    cy.get('#searchBox')
+      .clear()
+      .type(user.lastName);
+    cy.get('#searchBox')
+      .clear()
+      .type(user.email);
+    cy.get('#searchBox')
+      .clear()
+      .type(user.age);
+    cy.get('#searchBox')
+      .clear()
+      .type(user.salary);
+    cy.get('#searchBox')
+      .clear()
+      .type(user.department);
```

Comment:
```suggestion
```
you already do this searching in the test below

Reply:
Here I added the worker and below I edited worker

Reply:
yes, you have added the worker and then you assert that table contain added worker data. After that you start to search by columns, why do you need that?

Reply:
remove pls this code, it just takes test time but do nothing

Reply:
done